src/helics/core/UnknownHandleManager.hpp

```diff
@@ -67,6 +85,17 @@ class UnknownHandleManager {
     /** specify a found Source Filter*/
     std::vector<TargetInfo> checkForFilters(const std::string& newFilter) const;
 
+    /** specify a found input*/
+    std::vector<TargetInfo> checkForReconnectionInputs(const std::string& newInput) const;
+    /** specify a found input*/
+    std::vector<TargetInfo>
+        checkForReconnectionPublications(const std::string& newPublication) const;
+    /** specify a found input*/
+    std::vector<TargetInfo> checkForReconnectionEndpoints(const std::string& newEndpoint) const;
+
+    /** specify a found Source Filter*/
```

Comment:
Filters and Translators are technically managed as part of Cores, so reconnection of them doesn't make as much sense since they work asynchronously.  It wasn't entirely clear that reconnectable filters should be supported in the same way as endpoints, inputs, and publications.